Flepper.QueryBuilder/Base/BaseFlepperQueryBuilder.cs

```diff
@@ -1,12 +1,13 @@
-﻿using System;
+﻿using Flepper.QueryBuilder.Utils;
+using System;
```

Comment:
using System must be the first one

![image](https://user-images.githubusercontent.com/836496/30859312-803837a4-a299-11e7-80a7-e1a85825e6d2.png)